machine_management/configuring-hpa-for-an-application.adoc

```diff
@@ -0,0 +1,25 @@
+[id='configuring-hpa-for-an-application']
+= Configuring HPA for an application
+include::modules/common-attributes.adoc[]
+:context: application-hpa
+
+toc::[]
+
+You can configure Horizontal Pod Autoscaling (HPA) for an application that exports metrics.
```

Comment:
What is HPA? I think you need a conceptual module about it. You need to explain what it is, when you might use it, and what happens if you use it and cluster autoscaling.